File: ReactApp4.Server/Data/Player.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace ReactApp4.Server.Data
{
    public class Player
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("full_name")]
        public string? Full_name { get; set; }

        [Column("first_name")]
        public string? First_name { get; set; }

        [Column("last_name")]
        public string? Last_name { get; set; }

        [Column("is_active")]
        public bool? Is_active { get; set; }

        [Column("player_id")]
        public string? Player_id { get; set; }
    }
}

File: reactapp4.client/src/interfaces/Player.ts
interface Player {
    full_name: string;
    first_name: string;
    last_name: string;
    is_active: string;
    player_id: string;
}

interface SelectedPlayer {
    player_name: string;
    player_id: string;
    team_id: string;
    team_abbreviation: string;
}

export type { Player, SelectedPlayer };